__tests__/app.test.js

```diff
@@ -533,4 +533,49 @@ describe("DELETE /api/comments/:comment_id", () => {
   });
 });
 
-
+describe("GET /api/users", () => {
+  test("response code is 200", () => {
+    return request(app).get("/api/users").expect(200);
+  });
+  test("users table exists in database", () => {
+    return db
+      .query(
+        `SELECT EXISTS (
+                          SELECT FROM 
+                          information_schema.tables 
+                          WHERE 
+                          table_name = 'users'
+                          );`
+      )
+      .then(({ rows: [{ exists }] }) => {
+        expect(exists).toBe(true);
+      });
+  });
```

Comment:
There is no need to test if the users table exists as part of your tests for this endpoint, because it's more a part of the seeding process than it is strictly to do with this endpoint. And also, the fact that any of your other tests work means that the users table must exist - or where would they be getting the data from?